.ansible-lint

```diff
@@ -4,15 +4,6 @@ exclude_paths:
   - .github/
   - requirements.yml
   - src/**/cookiecutter/{{*
-  # - src/molecule_gce/cookiecutter/{{cookiecutter.molecule_directory}}
-
-skip_list:
-  # Temporary skips made during migration
-  - args[module]
```

Comment:
Please don't remove args[module] from skip_list. I can't comment for @ssbarnea but in its current form, this rule is kind of broken, at least with modules relying on an action plugin.

Reply:
Restored it 
Done